modules/core/index.js

```diff
@@ -0,0 +1,10 @@
+export { Connection } from './connection';
+export { Difference } from './difference';
+export { Entity } from './entity';
+export { Graph } from './graph';
+export { History } from './history';
+export { iD_Node as Node } from './node';
```

Comment:
Why is this one `iD_Node`?  Can it be just `Node` to be consistent with `Way` and `Relation`?